### PR TITLE
Improve the Stax UI following the Ledger Product team review

### PR DESCRIPTION
**Pull request by fbeutin-ledger**

### CODE REVIEW

File: src/ui/sign_tx_nbgl.c [hunk 1]
```diff
@@ -82,30 +83,44 @@ static dynamic_item_t
...
+#define TITLE_RETRACT_VOTE "Review transaction to\nretract vote"
+#define TITLE_CAST_VOTE    "Review transaction to\ncast vote"
+#define TITLE_SEND_NEO     "Review transaction to\nsend NEO"
+#define TITLE_SEND_GAS     "Review transaction to\nsend GAS"
+
+#define MAX4(a, b, c, d) MAX(MAX(a, b), MAX(c, d))
+// UPDATE THIS DEFINE if additional transaction types are added
+#define LONGEST_TITLE_SIZE \
+    MAX4(sizeof(TITLE_RETRACT_VOTE), sizeof(TITLE_CAST_VOTE), sizeof(TITLE_SEND_NEO), sizeof(TITLE_SEND_GAS))
+static char review_title[LONGEST_TITLE_SIZE];
```
[the remark this answers is not in the document]
Comment (fbeutin-ledger) [reply src/ui/sign_tx_nbgl.c:95]: Done

File: src/ui/sign_tx_nbgl.c [hunk 2]
```diff
@@ -73,6 +73,7 @@ typedef struct dynamic_slot_s {
 static void start_review(void);
 static void rejectUseCaseChoice(void);
 static nbgl_layoutTagValueList_t layout;
+static nbgl_pageInfoLongPress_t review_final_long_press;
 static nbgl_layoutTagValue_t current_pair;
 static nbgl_layoutTagValue_t static_items[MAX_NUM_STEPS + 1];
 static dynamic_slot_t dyn_slots[MAX_SIMULTANEOUS_DISPLAYED_SLOTS];
```
[the remark this answers is not in the document]
Comment (fbeutin-ledger) [reply src/ui/sign_tx_nbgl.c:79]: Done

File: src/ui/sign_tx_nbgl.c [hunk 3]
```diff
@@ -157,14 +172,6 @@ static void create_transaction_flow(void) {
...
-    .text = "Sign message on\nNeo N3 network?",
-    .icon = &C_icon_neo_n3_64x64,
-    .longPressText = "Hold to sign",
-    .longPressToken = 0,
-    .tuneId = TUNE_TAP_CASUAL,
-};
-
 static void review_final_callback(bool confirmed) {
     if (confirmed) {
         ui_action_validate_transaction(true, false);
```
[the remark this answers is not in the document]
Comment (fbeutin-ledger) [reply src/ui/sign_tx_nbgl.c:177]: Done

File: src/ui/menu.c [hunk 4]
```diff
@@ -132,14 +132,14 @@ enum {
 
 static bool settings_nav_callback(uint8_t page, nbgl_pageContent_t* content) {
     if (page == 0) {
-        content->type = SWITCHES_LIST;
-        content->switchesList.nbSwitches = NB_SETTINGS_SWITCHES;
-        content->switchesList.switches = (nbgl_layoutSwitch_t*) G_switches;
-    } else if (page == 1) {
         content->type = INFOS_LIST;
         content->infosList.nbInfos = ARRAY_COUNT(info_types);
         content->infosList.infoTypes = (const char**) info_types;
```
Comment (xchapron-ledger) [inline src/ui/menu.c:137]: cast not needed anymore, same just below.

**Review by xchapron-ledger — APPROVED** [review]

File: src/ui/menu.c (hunk 4 above)
Comment (fbeutin-ledger) [reply src/ui/menu.c]: Done

**Pull request merged**